src/Components/Admin/Scenes/scenecard.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { useHistory } from "react-router-dom";
+import defaultImage from "./defaultImage.svg";
+import Grid from "@material-ui/core/Grid";
+import IconButton from "@material-ui/core/IconButton";
+import MoreVertIcon from "@material-ui/icons/MoreVert";
+import MenuItem from "@material-ui/core/MenuItem";
+import Menu from "@material-ui/core/Menu";
+
+const useStyles = makeStyles(() => ({
+    card: {
+        fontSize: 16,
+    },
+    cardImage: {
+        width: "90%",
+        maxWidth: 400,
+        marginTop: 16,
+    },
+}));
+
+export default function SceneCard({ key, data }) {
+    const classes = useStyles();
+    const history = useHistory();
+    const [anchorEl, setAnchorEl] = React.useState(null);
+
+    const handleMenuClick = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleMenuClose = () => {
+        setAnchorEl(null);
+    };
+
+    const handleCardClick = () => {
+        history.push(`/admin/scene/${data.id}`);
+    };
+
+    const open = Boolean(anchorEl);
+
+    return (
```

Comment:
Hmm this looks extremely similar to the previous room card component. Is there a reason why we are not reusing it?

Reply:
We probably can, let me do that